modules/virt-creating-data-volumes-using-pvc-api.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-pvc-api_{context}"]
+
+= Creating data volumes using the PVC API
+
+When you create a data volume using the *pvc* API, the CDI creates the data volume based on your explicit:
+
+* `accessModes` (`ReadWriteOnce`, `ReadWrtieMany`, `ReadOnlyMany`)
+* `volumeMode` (`Filesystem` or `Block`)
+* `capacity` of `storage` (`5Gi`, for example)
+
+When you explicitly allocate a data volume by using the *pvc* API, you must consider file system overhead. File system overhead is the amount of space required by the CDI to manage the file. Failing to estimate file system overhead results in a storage allocation that is less than you requested.
```

Comment:
Failing to estimate file system overhead results in a storage allocation that is less than you requested.

I might write this as: "Failing to account for filesystem overhead in your storage capacity request may result in a PVC that is not large enough to accommodate your VM disk.

Reply:
Thanks, changed to:  _Failing to account for filesystem overhead in your storage capacity request may result in an underlying PVC that is not large enough to accommodate your virtual machine disk._